src/components/CreateList.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import getToken from '../lib/tokens';
+
+const CreateList = () => {
+  const newList = () => {
+    const newToken = getToken();
+    localStorage.setItem('newToken', newToken);
```

Comment:
I see that the other team is using 'token' as their token name so make sure to coordinate with them before merging :)